app/src/main/java/com/nervousfish/nervousfish/activities/WaitActivity.java

```diff
@@ -149,10 +140,19 @@ public void onNewDataReceivedEvent(final NewDataReceivedEvent event) {
 
     /**
      * Can be called by a button to cancel the pairing
+     *
      * @param view The view that called this method
      */
     public void cancelWaiting(final View view) {
         this.setResult(ConstantKeywords.CANCEL_PAIRING_RESULT_CODE);
         this.finish();
     }
+
+    private void goToMainActivity() {
```

Comment:
It's not needed, because this method won't show up in the documentation (because the method is private), so the documentation would be only for the programmer. Because the title is very, very clear I don't think it's needed, but well... Maybe the TA's like comments as well, so I'll do it